bip-0388.mediawiki

```diff
@@ -0,0 +1,306 @@
+<pre>
+  BIP: 388
+  Layer: Applications
+  Title: Wallet Policies for Descriptor Wallets
+  Author: Salvatore Ingala <salvatoshi@protonmail.com>
+  Comments-Summary: No comments yet.
+  Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-0388
+  Status: Draft
+  Type: Standards Track
+  Created: 2022-11-16
+  License: BSD-2-Clause
+  Post-History: 2022-05-10: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-May/020423.html
+</pre>
+
+== Abstract ==
+
+Wallet policies build on top of output descriptors to represent in a compact, easier to inspect way the types of descriptors that are typically used to represent "accounts" in a software wallet, or a hardware signing device. A wallet policy always represents exactly two descriptors, which produce the receive and change addresses that are logically part of the same account.
```

Comment:
Also, you use sometimes "output script descriptor", "output descriptor", and just "descriptor" to mean the same thing. Perhaps you should introduce it once as "output script descriptor (abbreviated with 'descriptor' in the following)".

Reply:
In 7d0c08e38acac3ef14095d0e8664c7332b7be381, I made sure that only the forms "output script descriptor" and "descriptor" appear. I think it's generally clear from context that both expression refers to the same.